### PR TITLE
Create robots file

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** Add robots.txt file with the proper link on vuepress sitemap 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-362275](https://oktainc.atlassian.net/browse/OKTA-362275)

### CODE REVIEW

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 1]
```diff
@@ -321,6 +321,26 @@ module.exports = {
         '/test_page/'
       ]
     },
+    'robots': {
+      host: 'https://developer.okta.com',
+      sitemap: "/docs-sitemap.xml",
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:326]: nit for consistency:
```suggestion
      sitemap: '/docs-sitemap.xml',
```

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 2]
```diff
@@ -321,6 +321,26 @@ module.exports = {
...
+      policies: [
+        {
+          userAgent: '*',
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:329]: do we also need to set 'crawl-delay: 10'? the existing `robots.txt` file has it

File: packages/@okta/vuepress-site/.vuepress/config.js (hunk 1 above)
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js]: done

File: packages/@okta/vuepress-site/.vuepress/config.js (hunk 2 above)
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:329]: fixed: https://60084b4343761827e3c7e7ad--reverent-murdock-829d24.netlify.app/robots.txt

**Review by shawyu-okta — APPROVED** [review]
🐧

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**